tcks/microprofile-reactive-messaging/pom.xml

```diff
@@ -59,6 +59,19 @@
             <groupId>org.eclipse.microprofile.reactive.messaging</groupId>
             <artifactId>microprofile-reactive-messaging-tck</artifactId>
             <version>${microprofile-reactive-messaging-api.version}</version>
+            <exclusions>
+                <!-- fix convergence: assertj is bumped to >= 3.19.0 via Quarkus BOM which is too new (see futher down) -->
+                <exclusion>
+                    <groupId>org.assertj</groupId>
+                    <artifactId>assertj-core</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!-- latest version of assertj that is known to work -->
```

Comment:
I couldn't find a better solution (apart from bumping assertj upstream).
Dependabot will most likely create PRs updating this dependency (which will obviously fail), since you cannot tell it to ignrore certain paths (AFAICS).

Reply:
@cescoffier @michalszynkiewicz any chance it could be updated in the TCK?

Reply:
Let me see, but I believe it's defined in the spec repository, so will take time (because that will not justify a release) to be updated.

Reply:
Ok, there is a weird breaking change in assertj. We need to update the TCK.

Reply:
the record, I've opened a PR on the spec to update the assertj version.